config/express.js

```diff
@@ -70,7 +70,7 @@ module.exports.http = {
       clientID: Reddit.data.clientID,
       clientSecret: Reddit.data.clientIDSecret,
       callbackURL: Reddit.data.redirectURL,
-      scope: "flair,modflair"
+      scope: "flair,modflair,modcontributors,wikiread,wikiedit,read,modposts"
```

Comment:
Yeah, I think we should look into it. I don't think it's **necessary** for this pull request to go in, but at least one person will start complaining as soon as she notices.